Sources/Sham/Models/StubRequest.swift

```diff
@@ -23,10 +23,12 @@ public struct StubRequest: Hashable, CustomStringConvertible {
     
     /// The string representation of the request.
     public var description: String {
-        if let method = self.method, let url = self.url {
-            return "\(method): \(url.absoluteString)"
-        } else if let url = self.url {
-            return "ALL: \(url.absoluteString)"
+        if let urlString = self.url?.sortedAbsoluteString {
+            if let method = self.method {
+                return "\(method): \(urlString)"
```

Comment:
This is why `POST` isn't working properly, it's using the enum name.

```suggestion
                return "\(method.rawValue): \(urlString)"
```